conbench/app/benchmarks.py

```diff
@@ -59,7 +64,7 @@ def show_benchmark_cases(bname: str) -> str:
         hardware_count_per_case=hardware_count_per_case,
         context_count_per_case=context_count_per_case,
         # cases=cases,
-        benchmark_result_count=len(results),
+        benchmark_result_count=len(matching_results),
```

Comment:
There was a naming confusion bug here, leading to a wrong count being displayed. (for loop iteration var leaked into out scope, typical Python thing)